README.md

```diff
@@ -35,13 +35,17 @@ dev_dependencies:
   flutter_launcher_icons: "^0.3.3"
   
 flutter_icons:
-  image_path: "icon/icon.png" 
+  image_path: "icon/icon.png"
+  image_path_android: "icon/icon_android.png"
+  image_path_ios: "icon/icon_ios.png"
   android: true
   ios: "Example-Icon"
 ```
 ```
 #### Attributes: 
 image_path: The location of the icon image file which you want to use as the app launcher icon
+image_path_android: The location of the icon image file specific for Android platform (optional)
```

Comment:
Perhaps include some further information like between line 50-54 that the users don't need all 3.